src/loki/polygon_search.cc

```diff
@@ -101,7 +91,7 @@ edges_in_rings(const google::protobuf::RepeatedPtrField<valhalla::Options_Ring>&
   for (const auto& ring_pbf : rings_pbf) {
     rings_bg.push_back(PBFToRing(ring_pbf));
     const ring_bg_t ring_bg = rings_bg.back();
-    rings_length += GetRingLength(ring_bg);
+    rings_length += bg::perimeter(ring_bg);
```

Comment:
how does this know to use spherical coordinates and to return meters?

Reply:
oh probably because of `geographic`?

Reply:
oops, I should've specified the `Haversine` function above, forgot that. But apparently it does some deductions from the underlying point type, no idea what that exactly defaults too though. So rather be explicit and put the Haversine strategy there.